examples/with-react-native-web/apps/native/tsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "expo/tsconfig.base",
+  "compilerOptions": {
+    "strict": true
+  }
+}
```

Comment:
Can we make the tsconfig shared? it doesn't make sense to share the babel config, but the tsconfig seems reasonable.

Reply:
Moved it out